scripts/update-arb-from-crowdin.mjs

```diff
@@ -266,17 +266,17 @@ function transformTranslations(data, locale, module, makeTemplate = false) {
     };
     let pluralString = '{count, plural,'
 
-    // add a zero quantity item if it doesn't exist
-    // to avoid translations like "0 players"
-    if (!plural.item.some((item) => item.$.quantity === 'zero')) {
-      const oneItem = plural.item.find((item) => item.$.quantity === 'one')
-      if (oneItem) {
-        plural.item.unshift({
-          $: { quantity: 'zero' },
-          _: oneItem._
-        })
-      }
-    }
+    //// add a zero quantity item if it doesn't exist
```

Comment:
You can delete the block instead of commenting it out.

Reply:
whoops, fixed